packages/config/config/devices/0x0438/4512748.json

```diff
@@ -0,0 +1,172 @@
+{
+  "manufacturer": "Namron",
+  "manufacturerId": "0x0438",  
+  "label": "4512748",
+  "description": "Awning Controller",
+  "devices": [
+    {
+      "productType": "0x0004",  
+      "productId": "0x53261"     
+    }
+  ],
+  "firmwareVersion": {
+    "min": "0.0",
+    "max": "255.255"
+  },
+  "associations": {
+    "1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+  },
+	"paramInformation": [
+		{
+			"#": "2",
+			"label": "Enable/Disable State Report Threshold",
```

Comment:
Avoid using "Enable/Disable" in labels. Usually there is a better label, or the options already imply that they are meant for enabling or disabling something.